static/docs/commands-reference/status.md

```diff
@@ -48,22 +48,47 @@ detected, `dvc status` prints this message:
     Pipeline is up to date. Nothing to reproduce.
 ```
 
-This says that no differences were detected, and therefore that no stages would
-be run again if `dvc repro` were executed.
+This indicates that no differences were detected, and therefore no stages would
+be run again by `dvc repro`.
 
 If instead, differences are detected, `dvc status` lists those changes. For each
-DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
-differ are listed. For each item listed, either the file name or the checksum is
-shown, and additionally a status word is shown describing the change:
+DVC-file (stage) with differences, the changes in _dependencies_ and/or
+_outputs_ that differ are listed. For each item listed, either the file name or
+the checksum is shown, and additionally a status word is shown describing the
+changes. This changes list provides a reference to both the status of a
+DVC-file, as well as the changes to individual dependencies and outputs
+described in it:
 
 - For the local workspace:
-  - _changed_ means the file has changed
+
+  - _changed checksum_ means that the update checksum of the data doesn't match
+    the one saved in the DVC-files;
+  - _always changed_ means that this is a special DVC-file with no dependencies
+    (orphans), which is considered always changed;
+  - _changed deps_ means that some changes in dependencies are incorporated in
+    the DVC-file, all these states are:
+    - _new_ : when reference of new dependency files are added into DVC-file;
+    - _modified_ : when checksum of an exsisting dependencies are changed in
+      workspace but not reflected in the DVC-file;
+    - _deleted_ : when an exsisting dependencies are removed from the workspace
+      but still referred in the DVC-file;
+    - _not in cache_ : when dependencies mentioned in DVC-file no longer exsist
+      in local cache;
+  - _changed outs_ means that some changes in output states are incorporated in
+    the DVC-file, all these states are:
+    - _new_ : when reference of new output files are are added into DVC-file;
+    - _modified_ : when checksum of an exsisting outputs are changed in
+      workspace but not reflected in the DVC-file;
+    - _deleted_ : when an exsisting outputs are removed from the workspace but
+      still referred in the DVC-file;
+    - _not in cache_ : when outputs mentioned in DVC-file no longer exsist in
+      local cache;
+
 - For comparison against a remote cache:
-  - _new_ means the file exists in the local cache but not the remote cache
-  - _deleted_ means the file doesn't exist in the local cache, but exists in the
-    remote cache
 
-For the _changed_ case, the `dvc repro` command is indicated.
+  - _new_ means the file exists in the local cache but not the remote cache;
+  - _deleted_ means the file doesn't exist in the local cache, but exists in the
+    remote cache;
```

Comment:
In all of the bullets above (starting in line 64), use consistent formatting:

- [x] All bullets already start with _italic_ term
- [ ] All bullets should either have column `: ` after the term OR the word ` means `, and then the description. Actually I vote for ` - ` instead, like in all the list of the existing command references.
- [ ] All bullets should be full sentences and end in period `.`  We no longer use semicolons `;` for this.

Thanks

Reply:
Also fixed in d0bbaba